manifests/install.pp

```diff
@@ -1,11 +1,14 @@
 # Install TFTP
-class tftp::install {
-  package { $::tftp::params::package:
+class tftp::install (
+  $package          = $tftp::params::package,
```

Comment:
Shouldn't the default value be `$::tftp::package`, not params, if the override from the main tftp class is to work?